packages/components/src/components/popover/popover-styles.spec.ts

```diff
@@ -2,14 +2,15 @@ import { getComponentCss } from './popover-styles';
 
 describe('getComponentCss()', () => {
   it.each<Parameters<typeof getComponentCss>>([
-    ['top', 'light'],
-    ['right', 'light'],
-    ['bottom', 'light'],
-    ['left', 'light'],
-    ['top', 'dark'],
-    ['right', 'dark'],
-    ['bottom', 'dark'],
-    ['left', 'dark'],
+    ['top', false, 'light'],
+    ['right', false, 'light'],
+    ['bottom', false, 'light'],
+    ['left', false, 'light'],
+    ['top', false, 'dark'],
+    ['right', false, 'dark'],
+    ['bottom', false, 'dark'],
+    ['left', false, 'dark'],
+    ['top', true, 'light'],
   ])('should return correct css for direction: %s and theme: %s', (...args) => {
```

Comment:
2nd parameter is missing in the description. Currently booleans are output for `theme`.